public/docs/_examples/structural-directives/dart/lib/unless_directive.dart

```diff
@@ -0,0 +1,31 @@
+// #docplaster
+// #docregion
+// #docregion unless-declaration
+import 'package:angular2/angular2.dart';
+// #enddocregion unless-declaration
+
+// #docregion unless-declaration
+@Directive(selector: '[myUnless]')
+class UnlessDirective {
+  // #enddocregion unless-declaration
+
+  // #docregion unless-constructor
+  TemplateRef _templateRef;
+  ViewContainerRef _viewContainer;
+
+  UnlessDirective(this._templateRef, this._viewContainer);
+  // #enddocregion unless-constructor
+
+  // #docregion unless-set
+  @Input() set myUnless(bool condition) {
```

Comment:
Did the formatter put @Input() on the same line (or care, either way)? I'm used to seeing annotations on their own lines.